Dockerfile

```diff
@@ -117,8 +116,8 @@ RUN Rscript -e "\
       paste0('$report_url', '$report_tag'), \
       paste0('$utils_url', '$utils_tag') \
     ); \
-  workflow_pkgs <- renv::dependencies('$WORKFLOW_DIR')[['Package']]; \
-  workflow_pkgs <- grep('^pacta[.]', workflow_pkgs, value = TRUE, invert = TRUE); \
+  workflow_pkgs <- pak::local_deps(root = '$WORKFLOW_DIR')[['ref']]; \
+  workflow_pkgs <- grep('^RMI-PACTA[/]|^local::.$', workflow_pkgs, value = TRUE, invert = TRUE); \
```

Comment:
looks awesome! Exactly what I had in mind :-)

Reply:
having to add `^local::.$` to prevent it from trying to install the "local package" is a bit weird, but yeah, otherwise works pretty good

Reply:
That's part of what my hesitancy was with adding a `DESCRIPTION` file to the workflow.* repo in the first place. 

You are essentially stating "this is an R package", so of course most tools will think of it now as a package.

Reply:
`pak` starts the dependency tree at the root package (every package is dependent on itself, it seems).